lib/utensils/custom_matchers.rb

```diff
@@ -146,6 +146,16 @@ def failure_message_when_negated
     def have_order(*args)
       Ordered.new(args)
     end
+
+    class HaveRole < Capybara::RSpecMatchers::HaveSelector
+      def initialize(role, *args, &filter_block)
+        super(:role, role, *args, &filter_block)
+      end
+    end
+
+    def have_role(*args)
```

Comment:
All other `have_` methods in this file are not taking a question mark, so I reused same convention.